src/Stoa.ts

```diff
@@ -774,21 +784,318 @@ class Stoa extends WebService
+
+        // Validating Parameter - page
+        if (req.query.page !== undefined) {
+            if (!Utils.isPositiveInteger(req.query.page.toString())) {
+                res.status(400).send(`Invalid value for parameter 'page': ${req.query.page.toString()}`);
+                return;
+            }
+            page = Number(req.query.page.toString());
+        }
+        else
+            page = 1;
+
+        this.ledger_storage.getBlockEnrollments(field, value, limit, page)
+        .then((data: any) => {
+            if ((data === undefined)) {
+                res.status(500).send("Failed to data lookup");
+                return;
+              }
+           else if (data.total_records === 0) {
+               return res.status(204).send(`The data does not exist. 'height': (${value})`);
+            }
+            else {
+                let enrollmentElementList: Array<IBlockEnrollmentElements> = [];
+                for (const row of data.enrollments) {
+                    enrollmentElementList.push(
+    {
+                            height: JSBI.BigInt(row.block_height).toString(),
+                            utxo:  new Hash(row.utxo_key, Endian.Little).toString(),
+                            enroll_sig:  new Hash(row.enroll_sig, Endian.Little).toString(),
+                            commitment:  new Hash(row.commitment, Endian.Little).toString(),
+                            cycle_length: row.cycle_length,
+                        }
+                    );
+                }
+                let enrollmentList: IBlockEnrollment =  {
+                    enrollmentElementList,
+                    total_data: data.total_records
+                };
+                return res.status(200).send(JSON.stringify(enrollmentList));
+            }
+        })
+        .catch((err) => {
+            logger.error("Failed to data lookup to the DB: " + err);
+           return res.status(500).send("Failed to data lookup");
+        });
+    }
+
+    /**
+     * GET /block-transactions
+     *
+     * Called when a request is received through the `/block-transactions` handler
+     * The parameter `height` is the height and `hash` is the hash of block
+     *
+     * @returns Returns transactions of block.
+     */
+     private getBlockTransactions(req: express.Request, res: express.Response) {
+        
+        let field: string;
+        let value: string | Buffer;
+        let limit: number;
+        let page: number;
+
+        logger.http(`GET /block-transactions/`);
+
+        // Validating Parameter - height
+        if (req.query.height !== undefined && Utils.isPositiveInteger(req.query.height.toString())) {
+            field = 'height';
+            value = String(req.query.height);
+        }
+        // Validating Parameter - hash
+        else if (req.query.hash !== undefined) {
+            field = 'hash';
+            const hash: string = String(req.query.hash);
+            value = new Hash(hash).toBinary(Endian.Little);
```

Comment:
Error handling is required.

Reply:
`  if (req.query.height !== undefined && Utils.isPositiveInteger(req.query.height.toString())) {
            field = 'height';
            value = String(req.query.height);
        }
        // Validating Parameter - hash
        else if (req.query.hash !== undefined) {
            field = 'hash';
            const hash: string = String(req.query.hash);
            value = new Hash(hash).toBinary(Endian.Little);
        }
        else {
            res.status(400).send(`Invalid value for parameter 'height': ${req.query.height} and 'hash': ${req.query.hash}`);
            return;
        }`

Reply:
error handling is there please review else block.